infra/README.md

```diff
@@ -0,0 +1,31 @@
+This is a work in progress to use Terraform to manage our cloud
+infrastructure.
+
+Instructions:
+
+- You will need a GCP project.  Create a service account for Terraform
+  with Editor role, and create a service account key and place it in
```

Comment:
The service account will also need the Service Networking Admin role, otherwise the google_service_networking_connection.private_vpc_connection creation will fail.

Reply:
I fixed this by making the terraform service account Owner instead of Editor.